### PR TITLE
Fix/ui explorer view

### PR DESCRIPTION
**Pull request by elfenheart**
- Fix default SQL query to get the current streamId or datasetId
- Change styling on view Queries to give more room to results
- Change body on Stream explore to limit to 80 characters
- Cursor change on View Queries table list
- Add selected queries styling to indicate which query is currently selected

explore

### CODE REVIEW

File: cdap-ui/app/features/datasets/datasets.less [hunk 1]
```diff
@@ -13,6 +13,20 @@ body.state-datasets-detail {
...
 
     .panel-body {
+
+      .queries {
+        font-size: 14px;
+
+        tr {
+          cursor: pointer;
+
+          &.current {
```
Comment (ajainarayanan) [inline cdap-ui/app/features/datasets/datasets.less]: Do we already have a style for hover?

File: cdap-ui/app/features/datasets/templates/tabs/explore.html [hunk 2]
```diff
@@ -95,18 +95,12 @@ <h4 class="panel-title" bs-collapse-toggle>
         </div>
 
         <!-- Right Panel -->
-        <div class="col-xs-6 well">
-          <div ng-show="responses.results.length">
-            <p><strong>{{ responses.request.status }}</strong></p>
+        <div class="col-xs-8 well">
+          <div ng-show="responses.request.status === 'FINISHED'">
```
Comment (ajainarayanan) [inline cdap-ui/app/features/datasets/templates/tabs/explore.html]: Shouldn't the ```well``` css class be in this div?

File: cdap-ui/app/features/datasets/templates/tabs/explore.html [hunk 3]
```diff
@@ -74,7 +74,7 @@ <h4 class="panel-title" bs-collapse-toggle>
       <div class="panel-body">
 
         <!-- Left Panel -->
-        <div class="col-xs-6">
+        <div class="col-xs-4 queries">
           <table class="table table-hover">
```
Comment (ajainarayanan) [inline cdap-ui/app/features/datasets/templates/tabs/explore.html]: Same here. Shouldn't the ```queries``` class be in the table?

File: cdap-ui/app/features/streams/streams.less [hunk 4]
```diff
@@ -49,6 +62,7 @@ body.state-streams-detail {
       }
 
       .results {
+        margin-top: 25px;
         div {
```
Comment (ajainarayanan) [inline cdap-ui/app/features/streams/streams.less]: I feel a css selector for all the divs under results css class is too generic. We should probably revisit this soon. This is the same for both datasets and streams. We should also be re-using them. Not sure why we aren't.

**Comment by ajainarayanan** [comment]
We are ideally having the same template and less file for both the datasets and streams explorer view. @elfenheart Any specific reason why we are replicating the code in multiple places? Was the view supposed to be different in datasets and streams?

**Comment by elfenheart** [comment]
@ajainarayanan yes, in that case a directive will make more sense. I will make the modification.

**Comment by ajainarayanan** [comment]
:+1: LGTM

**Pull request merged**